### ISSUE TITLE
Travis configuration does not define working environment variables

### ISSUE THREAD

**Issue by jhart-r7** [rapid7/sonar-client#15]
Without `SONAR_API_URL, SONAR_EMAIL`, and `SONAR_TOKEN`, the builds for PRs always fail.  Somehow master appears to succeed.

**Comment by pdeardorff-r7** [comment]
Yes I believe it's a config option  as a security setting.  I remember a checkbox somewhere that disallows these variables and checks for PRs.  I think the reasoning is anyone could submit a PR that reads these variables and sends them somewhere or prints them out.  Trying to think of a secure way to do that...
